infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/utils/RemoteHotRodServerRule.java

```diff
@@ -69,6 +69,7 @@ public void before() throws Exception {
 		// e.g. the usage as JUnit Rule concurrently with the usage as global test listener in Surefire.
 		synchronized ( running ) {
 			if ( running.compareAndSet( false, true ) ) {
+				RemoteHotRodServerHelper.beforeRun();
```

Comment:
this class is also used in integration tests module, where the server is in a different file system position